Options.lua

```diff
@@ -5395,8 +5395,10 @@ do
                                     type = "description",
                                     name = "These settings control how targets are counted when generating ability recommendations.\n\nBy default, the number of "
                                         .. "targets is shown on the bottom-right of the primary icon in the Primary and AOE displays, unless only one target is "
-                                        .. "detected.\n\n",
+                                        .. "detected.\n\n"
+                                        .. "Your true in-game target is always counted. Be aware that action-targeting mode does not work great, particularly with dot-based specs.\n\n",
```

Comment:
> Be aware that action-targeting mode does not work great, particularly with dot-based specs.

Revise:

"WARNING:  'Soft' targets from the Action Targeting system are not presently supported."